sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -924,3 +925,60 @@ def merge_accumulators(self, accumulators):
 
   def extract_output(self, accumulator):
     return accumulator[0]
+
+
+class ToString(object):
+  """
+  PTransform for converting a PCollection element, KV or PCollection Iterable
+  to string.
+  """
+
+  class Kvs(ptransform.PTransform):
+    """
+    Transforms each element of the PCollection to a string on the key followed
+    by the specific delimiter and the value.
+    """
+
+    def __init__(self, delimiter=None, **kwargs):
+      self.delimiter = delimiter or ","
+
+    def expand(self, pcoll):
+      input_type = KV[Any, Any]
+      output_type = List[str]
```

Comment:
Would not the output type can be just 'str' ? Same for other transforms.

Reply:
Yes, it can and you are right. As it is returning the string so output type should be str. I've fixed this. 
Thanks for letting me know.